docs/UserGuide.adoc

```diff
@@ -72,6 +72,12 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Listing all persons alphabetically by name : `sort`
+
+Shows a list of all persons, along with their non-private details, in the address book alphabetically by name. +
+Format: `list`
+
+
```

Comment:
Redundant blank line here :(

Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

Reply:
Same for other areas in the code below!